src/collidable.ts

```diff
@@ -0,0 +1,43 @@
+import { PhysObject } from './physObject';
+import { CanvasValues } from './constants';
+
+export class Collidable {
```

Comment:
Same comment about state-less classes: Just export the functions here directly. I'd also call this file something like collisionUtility.ts.

Reply:
done